bookkeeper-server/src/main/java/org/apache/bookkeeper/client/RegionAwareEnsemblePlacementPolicy.java

```diff
@@ -79,7 +80,7 @@ public class RegionAwareEnsemblePlacementPolicy extends RackawareEnsemblePlaceme
     protected Feature disableDurabilityFeature;
     private int lastRegionIndex = 0;
 
-    RegionAwareEnsemblePlacementPolicy() {
+    protected RegionAwareEnsemblePlacementPolicy() {
```

Comment:
why not change it to public?

Reply:
Change to protected so that we could extends `RegionAwareEnsemblePlacementPolicy` and implement some addtion policy in broker side in future.
IMO no need change to public.